csw-location/src/main/scala/csw/services/location/internal/LocationServiceImpl.scala

```diff
@@ -103,107 +76,67 @@ private[location] class LocationServiceImpl(cswCluster: CswCluster) extends Loca
-    *
-    * A `Future` is returned with empty list if no constant key is found for `LWWMap`.
-    *
-    * The returned `Future` will fail if list from `LWWMap` fails
-    */
+  //List all locations registered with CRDT
   def list: Future[List[Location]] = (replicator ? AllServices.get).map {
     case x@GetSuccess(AllServices.Key, _) => x.get(AllServices.Key).entries.values.toList
     case NotFound(AllServices.Key, _)     ⇒ List.empty
     case _                                => throw RegistrationListingFailed
   }
 
-  /**
-    * List all locations from `LWWMap` and complete the `Future` with `Location` values filtered on `ComponentType`.
-    *
-    * A `Future` is returned with empty list if no constant key is found for `LWWMap` or no locations are registered
-    * against the given `ComponentType`.
-    *
-    * The returned `Future` will fail if list from `LWWMap` fails
-    */
+  //List all locations registered for the given componentType
   def list(componentType: ComponentType): Future[List[Location]] = async {
     await(list).filter(_.connection.componentId.componentType == componentType)
   }
 
-  /**
-    * List all locations from `LWWMap` and complete the `Future` with `Location` values filtered on `Hostname`.
-    *
-    * A `Future` is returned with empty list if no constant key is found for `LWWMap` or no locations are registered
-    * against the given `Hostname`.
-    *
-    * The returned `Future` will fail if list from `LWWMap` fails
-    */
+  //List all locations registered with the given hostname
   def list(hostname: String): Future[List[Location]] = async {
     await(list).filter(_.uri.getHost == hostname)
   }
 
-  /**
-    * List all locations from `LWWMap` and complete the `Future` with `Location` values filtered on `ConnectionType`.
-    *
-    * A `Future` is returned with empty list if no constant key is found for `LWWMap` or no locations are registered
-    * against the given `ComponentType`.
-    *
-    * The returned `Future` will fail if list from `LWWMap` fails
-    */
+  //List all locations registered with the given connection type
   def list(connectionType: ConnectionType): Future[List[Location]] = async {
     await(list).filter(_.connection.connectionType == connectionType)
   }
 
-  /**
-    * Creates an `ActorRef` that subscribes for `Changed` messages for a given `Connection` from `LWWRegister` and pass
-    * it to [[akka.stream.scaladsl.Source]].
-    *
-    * The `Source` will then map the `Changed` event to [[csw.services.location.models.LocationUpdated]]
-    * if the `LWWRegister` contains the `Location` against `Connection`
-    *
-    * or to [[csw.services.location.models.LocationRemoved]] if there is no value against `Connection`.
-    *
-    * Un-track a given connection using [[akka.stream.KillSwitch]]
-    *
-    */
+  //Track the status of given connection
   def track(connection: Connection): Source[TrackingEvent, KillSwitch] = {
```

Comment:
Track should be explained well, what is killswitch, that it needs to be materialized before stream starts etc.